core/domain/user_services.py

```diff
@@ -685,12 +1123,24 @@ def _migrate_dashboard_stats_to_latest_schema(versioned_dashboard_stats):
 
 
 def get_current_date_as_string():
-    """Returns current date as a string of format 'YYYY-MM-DD'"""
+    """Gets current date.
+
+    Returns:
+        str. Current date as a string of format 'YYYY-MM-DD'.
+    """
     return datetime.datetime.utcnow().strftime(
         feconf.DASHBOARD_STATS_DATETIME_STRING_FORMAT)
 
 
 def parse_date_from_string(datetime_str):
+    """Gives parsed date from a string.
```

Comment:
Parses the given string, and returns the year, month and day of the date that it represents.